staging/src/k8s.io/apiserver/pkg/registry/rest/delete.go

```diff
@@ -39,6 +39,7 @@ type GarbageCollectionPolicy string
 const (
 	DeleteDependents GarbageCollectionPolicy = "DeleteDependents"
 	OrphanDependents GarbageCollectionPolicy = "OrphanDependents"
+	Unsupported      GarbageCollectionPolicy = "Unsupported"
```

Comment:
> Would you expect the garbage collector to delete a resource that doesn't support garbage collection as part of a cascade?

No.  Not supporting GC means not supporting GC.  The ownerrefs would not be respected and the GC  chain is lost when getting to that node in the graph.